src/app/core/services/notifications/message-handler.service.ts

```diff
@@ -46,14 +48,15 @@ export class MessageHandlerService {
   }
 
   triggerQuestionnaire(questionnaire: Assessment) {
-    return this.questionnaire
-      .pullDefinitionForSingleQuestionnaire(questionnaire)
-      .then(() =>
-        this.schedule.generateSingleAssessmentTask(
-          questionnaire,
-          AssessmentType.SCHEDULED,
-          Date.now()
-        )
+    return Promise.all([
+      this.appConfig.getReferenceDate(),
```

Comment:
What happens if app-config application is not used as part of the stack? is app-config a requirement for aRMT?

Reply:
No this is just the app config service within the app not the external app config.